### ISSUE TITLE
provide more information in the logs when insert check constraint fails

### ISSUE THREAD

**Issue by 0x777** [hasura/graphql-engine#1649]
Currently we throw this error message: `Check constraint violation. insert check constraint failed` which only hints at the problem. The logs should output the values of the row for which the constraint failed and the insert constraint as defined. This information along with the session variables should help debug the problem.

**Comment by dijam** [comment]
Any update on this ticket? :)

**Comment by tirumaraiselvan** [comment]
Related to #3658 3658

**Comment by macs91** [comment]
This would help a lot now

**Comment by dvasdekis** [comment]
Am having the same problem

**Comment by Manubi** [comment]
same with me.

**Comment by dylanesque** [comment]
Having this problem as well

**Comment by yash2code** [comment]
+1

**Comment by brunobraga95** [comment]
Same here

**Comment by rajat-sr** [comment]
Same problem

**Comment by Manubi** [comment]
If I remember correctly, most of the time it has to do with the permissions. So just got to the table and check if the authenticated user has permission to do things.
![Permissions_-_animal_vaccination_-_Data___Hasura](https://user-images.githubusercontent.com/20935138/100931242-8d25fa00-34ea-11eb-9793-2972a004eb8e.jpg)

**Comment by dylanesque** [comment]
Yes, but if the error message doesn't indicate that it's a permissions issue, it's unhelpful to developers newer to the platform.

**Comment by rosjerry** [comment]
any news on this issue?
is there some fix for it?
